### PR TITLE
feat: Jinja connected to fastAPI, added 'darkmoon', 'fastapi', and 't…

### PR DESCRIPTION
**Pull request by thulasiramanmt**
…emplating' to the workspace dictionary

# Pull request

**Resolves:** #7

### CODE REVIEW

**Review by Viraj-Ganguli — DISMISSED** [review]
Looks good! @DoodlePooge This seems ready for review.

File: darkmoon/app.py [hunk 1]
```diff
@@ -4,12 +4,24 @@
...
-from fastapi import FastAPI
+from fastapi import FastAPI, Request
+from fastapi.responses import HTMLResponse, Response
+from fastapi.templating import Jinja2Templates
 
 import darkmoon.api.v1.metadata.views as views
 from darkmoon.core.database import register_database
 from darkmoon.settings import Settings
 
+app = FastAPI()
```
Comment (tclasen) [inline darkmoon/app.py:15]: FastAPI app creation already happens later in this file, please remove this line

File: darkmoon/app.py [hunk 2]
```diff
@@ -4,12 +4,24 @@
...
+
+templates = Jinja2Templates(directory="templates")
```
Comment (tclasen) [inline darkmoon/app.py:17]: Please move the templates direction into the FastAPI application factory.

File: darkmoon/app.py [hunk 3]
```diff
@@ -4,12 +4,24 @@
...
+
+
+@app.get("/items/{id}", response_class=HTMLResponse)
```
Comment (tclasen) [inline darkmoon/app.py:20]: Please move this route into the FastAPI application factory below.

File: darkmoon/Jinja.py
Comment (tclasen) [inline darkmoon/Jinja.py:1]: Please delete this whole file, it isn't needed.

**Review by tclasen — CHANGES_REQUESTED** [review]

**Review by Viraj-Ganguli — APPROVED** [review]
Looks good, Pranav and I approve.

**Comment by DoodlePooge** [comment]
If the current code does work it is not clear on how it's supposed to work.

**Pull request closed without merging**